focus-group.js

```diff
@@ -43,6 +54,10 @@ function getToolbarItems(group) {
 }
 
 function isHorizontalOrientation(group) {
+  let focusGroupValue = group.getAttribute('focusgroup')
+  if (focusGroupValue === 'inline') return true
+  if (focusGroupValue === 'block') return false
```

Comment:
What is `focusgroup` has no `inline` or `block`? Some value should be by default?

Please, don’t rush. Read spec more careful and think twice and deeper.